src/Adapter/Import/Handler/ProductImportHandler.php

```diff
@@ -837,7 +837,7 @@ private function loadProductData(
 
             if ($productExistsByReference) {
                 $sqlPart .= 'p.`reference` = "' . pSQL($product->reference) . '"';
-            } elseif ($productExistsById) {
+            } else {
```

Comment:
Why? Is it really related to Symfony 4.4?

Reply:
It is related to phpstan :D